src/style.css

```diff
@@ -0,0 +1,33 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
```suggestion
```
it's forbidden to use anything except `box-sizing` in `*`

Reply:
How is it possible to reset the browser's default margin and padding in another way? It's not working in body or html.